code/modules/mob/living/bot/bot.dm

```diff
@@ -91,6 +91,7 @@
 		if(!locked)
 			open = !open
 			to_chat(user, "<span class='notice'>Maintenance panel is now [open ? "opened" : "closed"].</span>")
+			playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
```

Comment:
This should be refactored to the new tool sound system

Reply:
What does that look like?